### PR TITLE
docs: update (support and roadmap)

### PR DESCRIPTION
**Pull request by clemra**

### CODE REVIEW

File: pages/support.mdx [hunk 1]
```diff
@@ -9,26 +9,29 @@ description: "The Langfuse team and community can help you with questions and is
...
+
+## Email, Meeting and Slack
 
 For more sensitive or private matters, you can reach out to us directly. Please consider the public channels first, as they are more efficient for everyone.
 
-- Chat Widget (bottom right) on docs and Langfuse Cloud
 - Email via support@langfuse.com
 - [Schedule a Call](/schedule-demo) with one of the founders for sales or partnerships inquiries. Please provide a little context about why you'd like to speak.
-- Pro/Teams plan users: private Slack Connect channel with the Langfuse team. Request an invite via email/chat.
+- For: Pro/Teams plan users: private Slack Connect channel with the Langfuse team. Request an invite via support@langfuse.com
```
Comment (marcklingen) [inline pages/support.mdx]:
```suggestion
- For Pro/Teams plan users: private Slack Connect channel with the Langfuse team. Request an invite via support@langfuse.com
```

File: pages/support.mdx [hunk 2]
```diff
@@ -9,26 +9,29 @@ description: "The Langfuse team and community can help you with questions and is
...
-- [Feature Requests](/ideas) on GitHub Discussions.
-- [Bug reports](/issues) on GitHub Issues.
+- Our [documentation](/docs) is the best place to start looking for answers. It is comprehensive, and we invest significant time into maintaining it. You can also suggest edits to the docs via GitHub.
+- [Langfuse FAQs](/faq) on our website answer specific questions in more detail than the documentation can
 
-## Community channels
+## Support Channels
 
-- [Discord](/discord): Over 1,000 users are on our Discord Server. It's a great place to ask questions, share your projects, and get to know and get help from other Langfuse users.
+**- Ask any question in our [public Q&A](/gh-support) on GitHub Discussions.** Please include as much detail as possible (e.g. code snippets, screenshots, background information) as possible that will allow us to help you with your request.
```
Comment (marcklingen) [inline pages/support.mdx]:
```suggestion
- **Ask any question in our [public Q&A](/gh-support) on GitHub Discussions.** Please include as much detail as possible (e.g. code snippets, screenshots, background information) as possible that will allow us to help you with your request.
```

File: pages/docs/roadmap.mdx [hunk 3]
```diff
@@ -85,24 +85,28 @@ Subscribe to our mailing list to get occasional email updates about new features
...
 
 ## 🚧 In progress
 
-- Manual evaluations in Langfuse UI along multiple dimensions, currently only a single dimension is supported
-- Improved datasets UI/UX
-- Prompt management: multiple environments, comments on versions, fully-featured API
-- Infrastructure: queued ingestion to handle large and spiky loads on small instances
-- Improved tables across the Langfuse UI to display all relevant information and be more user-friendly
+- Prompt management: comments on versions, tags via SDK and api
+- Langfuse v3.0: async worker container, ingestion queues, OLAP database. Learn more in this [GitHub Discussion](https://github.com/orgs/langfuse/discussions/1902).
```
Comment (maxdeichmann) [inline pages/docs/roadmap.mdx]:
```suggestion
- Langfuse v3.0: preparing Langfuse for the next level of scale using an OLAP database, a queue and another container. Parts of it are already available in Langfuse Cloud and once migration is complete, self-hosting will be upgraded as well. Learn more in this [GitHub Discussion](https://github.com/orgs/langfuse/discussions/1902).
```

**Comment by marcklingen** [comment]
i'll take this from here. this divereged from main as this was not merged within 4 days!

**Pull request merged**